packages/link-list/src/js/react.js

```diff
@@ -50,7 +50,8 @@ export const AUlinkListItem = ({ text, link, linkComponent, li = {}, children, .
 	if( link ){
 		return (
 			<li { ...li }>
-				<LinkComponent { ...attributeOptions }>{ text }{ children }</LinkComponent>
+				<LinkComponent { ...attributeOptions }>{ text }</LinkComponent>
+				{ children }
```

Comment:
Now I'm super curious why that might be? As far as this looks like you're trying to add things outside the links inside the **link**list ...

We are going from:
```html
<ul>
	<li><a href="?">Text</a></li>
	<li><a href="?">Text</a></li>
	<li><a href="?">Text</a></li>
</ul>
```

to:
```html
<ul>
	<li>
		<a href="?">Headline</a>
		Text text text
	</li>
	<li>
		<a href="?">Headline</a>
		Text text text
	</li>
	<li>
		<a href="?">Headline</a>
		Text text text
	</li>
</ul>
```

I feel like the below list should be a new kind of list. But that's just me... :)

Reply:
So the text inside is a node, this allows people to add whatever they want inside. This change was required when we start nesting lists inside menus.

```
<ul>
	<li><a href="?">Text</a></li>
	<li><a href="?">Text</a></li>
	<li><a href="?">Text</a></li>
	<ul>
		<li><a href="?">Text</a></li>
		<li><a href="?">Text</a></li>
		<li><a href="?">Text</a></li>
	</ul>
</ul>
```

Reply:
The problem here though is that now link-lists won't allow icons inside your links...

Reply:
It should because the `text` property is a node. This means you can add anything inside the `a` tag yes?

Reply:
ok sure... I think that's probably ok. Just make sure your components don't do too much.

Reply:
Completely agree, thanks @dominikwilkowski 🏅